data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,114 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		cost 35100000
+		shields 26700
+		hull 19600
+		"required crew" 83
+		bunks 91
+		mass 870
+		drag 9.8
+		"outfit space" 887
+		"cargo space" 27
+		"weapon capacity" 364
+		"engine capacity" 150
+		"heat dissipation" 0.675
+		# Expensive features inherited from Ladybug-era technology, with some fuel (and improved heat dissipation):
+		"tactical scan power" 30
+		ramscoop 1.5
+		"fuel capacity" 800
+		weapon
+			"shield damage" 5000
+			"hull damage" 2500
+			# High hit force and wide radius for a spectacle if the leader dies.
+			# This should be adjusted downward if the ship is reused more widely.
+			"blast radius" 800
+			"hit force" 8000
+	outfits
+		"Thermal Repeater Rifle" 25
+		"Thermal Repeater Turret" 4
+		"Thunderhead Launcher" 6
+		"Thunderhead Missile" 320
+		"Thunderhead Storage Array" 4
+		"Warder Anti-Missile" 2
+		
+		"Liquid Sodium Cooler"
+		"Jump Drive"
+		"Quantum Keystone"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Outfits Expansion"
+		
+		"White Sun Reactor"
+		"Large Biochemical Cell"
+		
+		"Afterburner (Asteroid Class)"
+		"Reverser (Comet Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Planetary Class)"
+		
+		"Pulse Rifle" 65
+		"Fragmentation Grenades" 65
+		"Blade Of Might"
```

Comment:
I really think this sword ought to just wait for the actual Unfettered story instead of being an afterthought in this one.
Also, move the repeater rifles down here.

Reply:
But it would be visible at this point (and isnt the same sword btw)
9.17 will most likely not have the Uhai campaign and that will set the lore then
Could I still make him have one later on after that?

Reply:
> 9.17 will most likely not have the Uhai campaign and that will set the lore then

This is exactly why it should not be present here.

> Could I still make him have one later on after that?

I don't see why not. That's exactly what I'm suggesting you do.

Reply:
Done